File: lanacionscrape.py
from subprocess import call
import json, requests

r = requests.get('https://api-contenidos.lanacion.com.ar/json/V3/economia/cotizacionblue/DBLUE')

if r:
    data = r.json()
    call(["../bluelytics/add_blue.sh", data['compra'], data['venta'], 'invertir_online'])

File: lanacionscrape.js
// Initialization
var rURL = 'http://www.lanacion.com.ar/dolar-hoy-t1369';

var casper = require('casper').create({
    pageSettings: {
        loadImages:  false,        // We don't need images
        loadPlugins: false
    }
  });
var utils = require('utils');
var fs = require('fs');

//This function is our main, where we scrape the data
casper.start(rURL, function() {
  var dolarVenta = this.evaluate(function() {
      return document.querySelector('p#bventa').innerText.replace(',','.');
  });
  var dolarCompra = this.evaluate(function() {
      return document.querySelector('p#bcompra').innerText.replace(',','.');
  });

  var out = {'venta': dolarVenta, 'compra': dolarCompra};

  fs.write('lanacion_out.json', JSON.stringify(out), 'w');
});


casper.run();